servicebus/data-plane/azure-servicebus/src/main/java/com/microsoft/azure/servicebus/MessageBody.java

```diff
@@ -21,17 +21,17 @@
     private List<List<Object>> sequenceData;
     private List<byte[]> binaryData;
     
-    private MessageBody() {}
+    private MessageBody() {
+
```

Comment:
This extra line is unnecessary i believe. having the } on a new line is sufficient.